README.md

```diff
@@ -73,16 +73,89 @@ for the word "anonymous"
 https://www.google.com/webhp?#q=https://www.uservoice.com:+anonymous
 we don't see any results in their product.
 But there is something on their forum: https://feedback.uservoice.com/forums/1-product-management/suggestions/956361-allow-anonymous-users-to-create-ideas-without-leav
++ ***Sayat.me***: https://www.sayat.me/ this is a feedback tool that is used
+by individuals to gain insight into how people honestly feel about them. It has
+a good interface for the feedback itself but it's too specific which means that
+it doesn't apply to organisations such as dwyl. The feedback is public and
+comments can be made on the feedback:
+![sayat.me feedback](https://cloud.githubusercontent.com/assets/12450298/24554452/38df2670-15f3-11e7-9cdd-8757072183e0.png)
```

Comment:
@jackcarlisle thanks for [uploading these images to an issue](https://github.com/dwyl/feedback/issues/6) to save on storing them in the repo. 👍 
However it has highlighted that including lots of screenshots in the main readme means the page will take an inordinately long time to load on a "_slower_" internet connection ...
![feedback-page-load-time-over-3g-36sec](https://cloud.githubusercontent.com/assets/194400/24566565/dd7befca-1651-11e7-95a8-78e4e624a48e.png)
On a "Regular 3G connection" the readme will take 38 seconds to load ... ⏳ 

> Should we consider creating a `research.md` file for this content so that it's not loaded for the _main_ readme? 

P.S. I know I'm the _worst_ for including images in readmes, so I'm going to _change_ that for all tutorials I contribute to from now on. 👍